pkg/commands/process/settings/rules/javascript/lang/weak_encryption/.snapshots/TestJavascriptWeakEncryptionSummary-summary_javascript_lang_weak_encryption_md5.js

```diff
@@ -0,0 +1,25 @@
+critical:
+    - policy_name: ""
```

Comment:
@cfabianski  Is it ok that this policy names are empty?
any idea from where this comes from?

Reply:
ruby ones also have this field empty.

Reply:
ok